modules/runner/src/integrationTest/java/org/apache/ignite/internal/AbstractClusterIntegrationTest.java

```diff
@@ -82,7 +82,7 @@ public abstract class AbstractClusterIntegrationTest extends BaseIgniteAbstractT
 
     /** Work directory. */
     @WorkDirectory
-    protected Path workDir;
+    protected static Path workDir;
```

Comment:
Hm, this changes the lifecycle of work directory. Before that it was created for each test and cleaned after each test. Now it will be created once before all tests. Are you sure this works ok?